src/Components/ShareProfile.js

```diff
@@ -7,9 +7,10 @@ import { Toast } from 'primereact/toast'
 export default function ShareProfile({ username }) {
   const [show, setShow] = useState(false)
   const toast = useRef(null)
+  const profileUrl = `${location.protocol}//${location.hostname}/${username}`
```

Comment:
Why not just, location.href? 🤔 

```suggestion
  const profileUrl = location.href
```

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.svg)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question

Reply:
Definitely a good idea! Will make the URLs work with #1210 too 👍